tests/reference/ast-escape_character1-eb0f575.stdout

```diff
@@ -0,0 +1 @@
+(Module [(FunctionDef testEscapeCharacters ([] [] [] [] [] [] []) [(AnnAssign (Name s1 Store) (Name str Load) (ConstantStr "Hello	World" ()) 1) (AnnAssign (Name s2 Store) (Name str Load) (ConstantStr "Hello    World" ()) 1) (AnnAssign (Name s3 Store) (Name str Load) (ConstantStr "HelloWorld" ()) 1) (Expr (Call (Name print Load) [(Name s1 Load)] [])) (Expr (Call (Name print Load) [(Name s2 Load)] [])) (Expr (Call (Name print Load) [(Name s3 Load)] []))] [] () ()) (FunctionDef main0 ([] [] [] [] [] [] []) [(Expr (Call (Name testEscapeCharacters Load) [] []))] [] () ()) (Expr (Call (Name main0 Load) [] []))] [])
```

Comment:
@certik `"Hello World"` and `"HelloWorld"` make sense?

Reply:
When I print the AST locally , I get the following (which seems like something we would expect) ... not sure what the `""` character stands for 
```
(lf) anutosh491@spbhat68:~/lpython/lpython$ ./src/bin/lpython --show-ast tests/escape_character1.py --indent
(Module
    [(FunctionDef
        testEscapeCharacters
        ([]
        []
        []
        []
        []
        []
        [])
        [(AnnAssign
            (Name
                s1
                Store
            )
            (Name
                str
                Load
            )
            (ConstantStr
                "Hello  World"
                ()
            )
            1
        )
        (AnnAssign
            (Name
                s2
                Store
            )
            (Name
                str
                Load
            )
            (ConstantStr
                "Hello   World"
                ()
            )
            1
        )
        (AnnAssign
            (Name
                s3
                Store
            )
            (Name
                str
                Load
            )
            (ConstantStr
                "Hello
                      World"
                ()
            )
            1
        )
        (Expr
            (Call
                (Name
                    print
                    Load
                )
                [(Name
                    s1
                    Load
                )]
                []
            )
        )
        (Expr
            (Call
                (Name
                    print
                    Load
                )
                [(Name
                    s2
                    Load
                )]
                []
            )
        )
        (Expr
            (Call
                (Name
                    print
                    Load
                )
                [(Name
                    s3
                    Load
                )]
                []
            )
        )]
        []
        ()
        ()
    )
    (FunctionDef
        main0
        ([]
        []
        []
        []
        []
        []
        [])
        [(Expr
            (Call
                (Name
                    testEscapeCharacters
                    Load
                )
                []
                []
            )
        )]
        []
        ()
        ()
    )
    (Expr
        (Call
            (Name
                main0
                Load
            )
            []
            []
        )
    )]
    []
)
```

Reply:
I think the AST and ASR should store the escape sequences info in the form of `\t`, `\v`, ... and then handle them later in the backends. CPython does the same thing:
```clojure
python -m ast examples/expr2.py
Module(
   body=[
      Expr(
         value=Constant(value='Hello\tWorld')),
      Expr(
         value=Constant(value='Hello  \x08World')),
      Expr(
         value=Constant(value='Hello\x0bWorld'))],
   type_ignores=[])
```

Reply:
Note sure , how I could address the above comment @Thirumalai-Shaktivel . Locally my thing doesn't store these escape sequences !

Reply:
@Thirumalai-Shaktivel As we have tested out and merged #1502 , I can see that the AST now returns something which is similar to what CPython returns . I'll resolve conflicts here and modify the tests/references accordingly for the same . Then this could go in !

Reply:
I've updated the referneces accordingly . Should be good to go once the tests pass !